velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2858,41 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+TEST_F(DateTimeFunctionsTest, currentTimestampTest) {
+  const auto evaluateCurrentTimestamp = [&](int64_t expectedTimestamp,
+                                            const std::string& timeZone) {
+    VectorPtr expected =
+        makeTimestampWithTimeZoneVector(expectedTimestamp, timeZone.c_str());
+    VectorPtr actual =
+        evaluateWithTimestampWithTimezone("current_timestamp", 0, "+00:00");
+
+    // Accept 5 milliseconds lag since sometimes the machine is slow.
+    ASSERT_TRUE(actual->compare(expected.get(), 0, 0) < 5)
+        << "at " << 0 << ": expected " << expected->toString(0) << ", but got "
```

Comment:
The "at" << 0 << is not needed since the usage of vectors is for the implementation of the test, not the logic of the test. 

Same for the function below.